src/resolver/resolver.zig

```diff
@@ -2136,8 +2136,24 @@ pub const Resolver = struct {
             }
         }
 
-        // Mostly to cut scope, we don't resolve `NODE_PATH` environment variable.
-        // But also: https://github.com/nodejs/node/issues/38128#issuecomment-814969356
+        // resolve `NODE_PATH`
+        var node_path: []const u8 = "";
+        if (r.env_loader) |env_loader| {
+            node_path = env_loader.get("NODE_PATH") orelse "";
```

Comment:
couldn't this just be `const node_path =` and then moving the second if check inside of this one?

Reply:
All done